src/Pos/Launcher/Runner.hs

```diff
@@ -358,13 +360,15 @@ runCH params@NodeParams {..} sscNodeContext act = do
             { ncJLFile = jlFile
             , ncSscContext = sscNodeContext
             , ncBlkSemaphore = semaphore
-            , ncLrcSync = lrcSync
+            , ncLrcContext = LrcContext
```

Comment:
Seems like `RecordWildCards` can be used here to simplify this constructor into `let ctx = NodeContext{..}`